tests/test_utils/gcp_system_helpers.py

```diff
@@ -172,3 +173,29 @@ def grant_bucket_access(cls, bucket: str, account_email: str):
                 bucket_name,
             ]
         )
+
+    @classmethod
+    def delete_secret(cls, name: str, silent: bool = False):
+        cmd = ["gcloud", "secrets", "delete", name, "--project", GoogleSystemTest._project_id(), "--quiet"]
+        cls.execute_with_ctx(cmd, key=GCP_SECRET_MANAGER_KEY, silent=silent)
+
+    @classmethod
+    def create_secret(cls, name: str, value: str):
+        with tempfile.NamedTemporaryFile(delete=False) as tmp:
+            tmp.write(value.encode("UTF-8"))
+        cmd = ["gcloud", "secrets", "create", name,
+               "--replication-policy", "automatic",
+               "--project", GoogleSystemTest._project_id(),
+               "--data-file", tmp.name]
+        cls.execute_with_ctx(cmd, key=GCP_SECRET_MANAGER_KEY)
+        os.remove(tmp.name)
```

Comment:
Does this work?

```suggestion
        with tempfile.NamedTemporaryFile() as tmp:
            tmp.write(value.encode("UTF-8"))
            tmp.flush()
            cmd = ["gcloud", "secrets", "create", name,
                   "--replication-policy", "automatic",
                   "--project", GoogleSystemTest._project_id(),
                   "--data-file", tmp.name]
            cls.execute_with_ctx(cmd, key=GCP_SECRET_MANAGER_KEY)
```

(otherwise in case of error the file leaks.)

Reply:
execite_context never fails. But yeah. This way is nicer.

Reply:
Yep. This works, and is more robust.

Reply:
Looks like you changed update_secret, but not create_secret. Could you change this one too please?

Reply:
Bad me! Fixed.